src/CacheItemSelector/CacheItemSelector.php

```diff
@@ -35,6 +36,14 @@ public function forget(): void
     {
         collect($this->urls)
             ->map(function ($uri) {
+                if (
+                    is_string($uri) &&
```

Comment:
I don't understand what this does. Could you refactor this to a method with a name that explains what it does? Don't use `&&` but early returns in that method.

Reply:
thank you for your comment, I add 'isUriHashed' to explain that more clear
@freekmurze